deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -0,0 +1,278 @@
+          createRetryableTicketGasParams.maxSubmissionCost, // uint256 maxSubmissionCost,
+          refundAddress,                                    // address excessFeeRefundAddress,
+          refundAddress,                                    // address callValueRefundAddress,
+          createRetryableTicketGasParams.gasLimit,          // uint256 gasLimit,
+          createRetryableTicketGasParams.maxFeePerGas,      // uint256 maxFeePerGas,
+          l2ProposalData,                                   // bytes calldata data
+        ],
+        value: createRetryableTicketGasParams.deposit
+      },
+      // 2. Approve the COMP gateway to take Timelock's COMP for bridging
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [compGatewayAddress, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from mainnet to Arbitrum rewards
+      {
+        contract: arbitrumL1GatewayRouter,
+        signature: 'outboundTransferCustomRefund(address,address,address,uint256,uint256,uint256,bytes)',
+        args: [
+          COMP.address,                             // address _token,
+          refundAddress,                            // address _refundTo,
+          rewards.address,                          // address _to,
+          COMPAmountToBridge,                       // uint256 _amount,
+          compGasParams.gasLimit,                   // uint256 _maxGas,
+          compGasParams.maxFeePerGas,               // uint256 _gasPriceBid,
+          utils.defaultAbiCoder.encode(
+            ['uint256', 'bytes'],
+            [compGasParams.maxSubmissionCost, '0x']
+          )                                         // bytes calldata _data
+        ],
+        value: compGasParams.deposit
+      },
+      // 4. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarketsJSON)]
+        )
+      },
+      // 5. Displace v2 USDT COMP rewards
+      {
+        contract: comptrollerV2,
+        signature: '_setCompSpeeds(address[],uint256[],uint256[])',
+        args: [
+          [cUSDTAddress],
+          [0],
+          [0],
+        ],
+      },
+    ];
+
+    const description = "# Configurate Arbitrum cUSDCNv3 market, set reward config, USDC native and COMP, and update ENS text record.";    
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(mainnetActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  // async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
```

Comment:
You can just delete this, not comment it out

Reply:
Ok 👍